DataFormats/HcalDetId/src/HcalCalibDetId.cc

```diff
@@ -33,6 +33,18 @@ HcalCalibDetId::HcalCalibDetId(int ieta, int iphi) : HcalOtherDetId(HcalCalibrat
       |(((ieta > 0)?(1):(0))<<11); // z side, bit [11]
 }
 
+HcalCalibDetId::HcalCalibDetId(CalibDetType dt, int value) : HcalOtherDetId(HcalCalibration) {
+  id_|=(dt<<17);
+  id_|=value&0xFF;
```

Comment:
is the value fixed per channel and never changes between events?
DetId can only be used as a channel or detector cell/substructure index.